src/async/async_api.c

```diff
@@ -43,7 +43,7 @@ __async_get_format(WT_CONNECTION_IMPL *conn, const char *uri,
 	 * is a possibility a duplicate entry might be inserted, but
 	 * that is not harmful.
 	 */
-	TAILQ_FOREACH(af, &async->formatqh, q) {
+	TAILQ_FOREACH (af, &async->formatqh, q) {
```

Comment:
I've listed these macros under the `ForEachMacros` list in Clang Format (which is why it indents nicely).
Unfortunately, a side effect of this is what it treats `TAILQ_FOREACH` like `for` and puts a space after it.